modules/models/mentorship_models.py

```diff
@@ -0,0 +1,166 @@
+from pydantic import Field, BaseModel
+from datetime import datetime
+from typing import Union
+
+from modules.models.shared_models import AirtableRowBaseModel, AirtableUser
+
+
+class MentorshipService(AirtableRowBaseModel):
+    name: str = Field(
+        ..., example="Pair Programming", description="Name of the service"
+    )
+    slug: str = Field(
+        ...,
+        example="pair_programming",
+        description="Snake cased value for the service, used for identification and other purposes",
+    )
+    description: str = Field(
+        ...,
+        example="Work on a programming problem with a mentor while on a call",
+        description="Description of the service",
+    )
+
+
+class MentorshipSkillset(AirtableRowBaseModel):
+    name: str = Field(
+        ..., example="Pair Programming", description="Name of the service"
+    )
+    slug: str = Field(
+        ...,
+        example="pair_programming",
+        description="Snake cased value for the service, used for identification and other purposes",
+    )
+    mentors: list[str] = Field(
+        None,
+        example="['recoakW045JkGgQB7', 'rec9Un0YIvPsFjPZh', 'recnfnbHDZdie8jcD']",
+        description="List of Airtable record IDs for mentors that have this skillset",
```

Comment:
Now that we know exactly which columns and tables are relevant, once we decommission the old pybot, I would love to see Airtable cleaned up to be more atomic. We have the `Skillsets` table and then the `Skillsets` column in the mentors table, and I would love to see only one. I'm sure there's other data issues that are more solvable with this rewrite, so when we find those opportunities we should note them somewhere

Reply:
Yeah I actually tried to do this our new Airtable. If we want to maintain the relationships, we either have to do it this way or set up a separate table with a `skillset_id` and `mentor_id`. We could also just go to straight names and then search for those, but that makes it harder to keep it up to date.